src/tools/cephfs/Dumper.h

```diff
@@ -31,10 +31,16 @@ class Dumper : public MDSUtility {
 private:
   mds_role_t role;
   inodeno_t ino;
+  bool inited;
 
 public:
-  Dumper() : ino(-1)
+  Dumper() : ino(-1), inited(false)
   {}
+  ~Dumper() {
```

Comment:
Perhaps this should be done in `MDSUtility::~MDSUtility`?

Reply:
great! thanks, it can be done